src/jit/gentree.h

```diff
@@ -4097,6 +4101,30 @@ inline bool GenTree::IsZero()
     return false;
 }
 
+// Note that, unlike IsZero(), this only returns true only for integral types.
```

Comment:
(Minor) word "only" got repeated twice.

".. this only trues true for integral types"

Similar correction is needed on line 4116